config/settings/production.py

```diff
@@ -85,7 +85,11 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#default-from-email
 DEFAULT_FROM_EMAIL = env(
     "DJANGO_DEFAULT_FROM_EMAIL",
-    default="Grupa Zasoby <noreply@grupazasoby.pl>",
+    default="Grupa Zasoby <powiadomienia@grupazasoby.pl>",
+)
+REPLY_TO_EMAIL = env(
```

Comment:
Is this overwritten in production environment? @tadeoos

Reply:
not yet

Reply:
That's good. If it were my change wouldn't have any effect. So I'm happy that it's not.